### PR TITLE
Added function 'AddRequestHeader'

### PR DESCRIPTION
**Pull request by gsantosdasilva**

### CODE REVIEW

**Comment by gsantosdasilva** [comment]
I need to finish this PR, its not complete.

**Comment by silvath** [comment]
The API "~/Data/GetCookieValues?useHeaders=true should work in Docs

File: src/WebDocs/wwwroot/app/functions/AddRequestHeader/samples/001/content.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="/drapo.js"></script>
+    <title>Function - AddRequestHeader</title>
+</head>
+<body>
+    <span>Function - AddRequestHeader</span>
+    <div d-dataKey="data" d-dataType="object" d-dataProperty-url="~/api/Sample/GetCookieValues?useHeaders=true"></div>
```
[the remark this answers is not in the document]
Comment (gsantosdasilva) [reply src/WebDocs/wwwroot/app/functions/AddRequestHeader/samples/001/content.html:9]: Okay, in commit "GetCookieValuesMethod", 
I added the API that should work in the sample, however there is no object in the "content.html", so the sample returned exactly a list, as requested, "url - {path to api}".

I added the same method in the Controller and changed the html path.

**Comment by silvath** [comment]
The api in controller is never called

**Pull request merged**